### PR TITLE
Translate Cookbook index.md

### PR DESCRIPTION
**Pull request by Pwntus**
Ready to be merged.

### CODE REVIEW

File: src/v2/cookbook/index.md [hunk 1]
```diff
@@ -1,91 +1,91 @@
...
 ---
 
-## The Cookbook vs the Guide
+## Kokboken vs guiden
 
-How is the cookbook different from the guide? Why is this necessary?
+Hur skiljer sig kokboken från guiden? Varför behövs den?
 
-* **Greater Focus**: In the guide, we're essentially telling a story. Each section builds on and assumes knowledge from each previous section. In the cookbook, each recipe can and should stand on its own. This means recipes can focus on one specific aspect of Vue, rather than having to give a general overview.
+* **Strörre fokus**: I guiden berättar vi en historia. Varje sektion bygger på och förutsätter kunnskap från tidigare sektioner. I kokboken kan (och borde) varje recept stå för sig själv. Detta betyder att recept kan fokusera på en specifik aspekt i Vue istället för att ge en allmän överblick.
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: strörre :arrow_right: större

File: src/v2/cookbook/index.md [hunk 2]
```diff
@@ -1,91 +1,91 @@
...
+* **Lär JavaScript**: I guiden förväntas åtminstone medelmåttig ES5 JavaScript kunskap. Vi förklarar t.ex. inte hur `Array.prototype.filter` fungerar i en computed property (beräknad egenskap) som filtrerar en lista. I kokboken kan dock grundläggande JavaScript-funktioner (ES6/2015+ inkluderat) utforskas och förklaras i kontexten av hur det hjälper oss att bygga bättre Vue-applikationer.
 
-* **Exploring the Ecosystem**: For advanced features, we assume some ecosystem knowledge. For example, if you want to use single-file components in Webpack, we don't explain how to configure the non-Vue parts of the Webpack config. In the cookbook, we have the space to explore these ecosystem libraries in more depth - at least to the extent that is universally useful for Vue developers.
+* **Utforska ekosystemet**: För avancerade funktioner förväntas kunskap om ekosystemet. T.ex. om du vill använda enkla komponentfiler i Webpack så förklarar vi inte hur du konfigurerar inställningar som inte är Vue-relaterade i Webpack. I kokboken har vi strörre utrymme att utforska dessa ekosystembibliotek i djupet - åtminstone till den grad det är övervägande användbart för Vue-utvecklare.
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: `strörre` :arrow_right: `större`

File: src/v2/cookbook/index.md [hunk 3]
```diff
@@ -1,91 +1,91 @@
...
+<p class="tip">Vänligen notera att även med alla dessa skillnader så är kokboken inte en steg-för-steg manual. Största delen av innehållet förväntar att du ha baskunskap i koncept så som HTML, CSS, JavaScript, npm/yarn, etc.</p>
 
-## Cookbook Contributions
+## Bidra till kokboken
 
-### What we're looking for
+### Vad vi letar efter
 
-The Cookbook gives developers examples to work off of that both cover common or interesting use cases, and also progressively explain more complex detail. Our goal is to move beyond a simple introductory example, and demonstrate concepts that are more widely applicable, as well as some caveats to the approach.
+Kokboken ger utvecklare exempel som kan jobbas vidare på och som behandlar både vanliga och intressanta användningsområden, men även progressivt förklarar komplex detalj. Vårt mål är att förbigå ett simpelt introduktionsexempel och demonstrera koncept som har bredare användningsområden, men även varningar att se upp för.
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: Till synes har den engelska översättningen fel - det bör väl stå `details` alltså blir det `detaljer` här, eller?

File: src/v2/cookbook/index.md [hunk 4]
```diff
@@ -1,91 +1,91 @@
...
 
-If you're interested in contributing, please initiate collaboration by filing an issue under the tag **cookbook idea** with your concept so that we can help guide you to a successful pull request. After your idea has been approved, please follow the template below as much as possible. Some sections are required, and some are optional. Following the numerical order is strongly suggested, but not required.
+Om du är intresserad av att bidra, var då vänligen starta med att skapa en issue under taggen **cookbook idea** där du inkluderar ditt koncept så att vi kan hjälpa till med att utföra en vällyckad pull request. Efter att din idé har blivit godkänd, vänligen följ mallen nedan så noggrant som möjligt. Några sektioner är obligatoriska och några är valfria. Det rekommenderas starkt att följa den numeriska ordningen, med det krävs inte.
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: `var då vänligen starta` :arrow_right: `var då vänlig och starta` eller `starta vänligen`, tycker jag?

File: src/v2/cookbook/index.md [hunk 5]
```diff
@@ -1,91 +1,91 @@
...
-> * Introduce complexities one at a time
-> * Link to other docs, rather than re-explaining concepts
-> * Describe the problem, rather than assuming familiarity
-> * Explain the process, rather than just the end result
-> * Explain the pros and cons of your strategy, including when it is and isn't appropriate
-> * Mention alternative solutions, if relevant, but leave in-depth explorations to a separate recipe
+> * Lösa ett specifikt, vanligt problem
+> * Starta med det enklaste möjliga exemplet
+> * Introducera komplexitet en gång i taget
+> * Länka till andra dokument instället för att förklara på nytt
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: `instället` :arrow_right: `i stället`

File: src/v2/cookbook/index.md [hunk 6]
```diff
@@ -1,91 +1,91 @@
...
+> * Beskriva problemet, kunskap antas inte
+> * Förklara processen, ge inte endast slutresultatet
+> * Förklara för- och nackdelar med din strategi men även när den (inte) är lämplig
```
Comment (sustained) [inline src/v2/cookbook/index.md]: Now I feel like I am nitpicking but seems easier to just write:

:question: `även när den (inte) är lämplig` :arrow_right: `även när den är (o)lämplig`

Feel free to ignore this one.

File: src/v2/cookbook/index.md [hunk 7]
```diff
@@ -1,91 +1,91 @@
...
+> * Nämna alternativa lösningar (om relevanta sådana), men lämna fördjupad utforskning till ett separat recept
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :question: The `sådana` feels out of place to me but maybe that is a thing and I just haven't seen it used that way.

If it really feels right to you then please feel free to ignore this one.

File: src/v2/cookbook/index.md [hunk 8]
```diff
@@ -1,91 +1,91 @@
...
 
-We request that you follow the template below. We understand, however, that there are times when you may necessarily need to deviate for clarity or flow. Either way, all recipes should at some point discuss the nuance of the choice made using this pattern, preferably in the form of the alternative patterns section.
+Vi ber att du följer mallen nedan. Vi förstår dock att det kan uppstå tillfällen då du kan vara tvungen att avvika för att förtydliga eller för att få texten att flöda. Hursomhelst borde alla recept vid något tillfälle diskutera nyanser med gjorda val genom att följa dessa mallar, helst genom exemplet på alternativa mönstersektionen.
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :question: `då` :arrow_right: `när`, jag vet att den här användningen av `då` anses ålderdomligt i tal (där `när` föredras starkt) men vet också att det är lite vanligare i skrift således är det nog inte världens största problem.

File: src/v2/cookbook/index.md [hunk 9]
```diff
@@ -1,91 +1,91 @@
...
 
-### When To Avoid This Pattern
+### När detta mönster bör undvikas
 
-_optional_
+_valfri_
 
-This section is not required, but heavily recommended. It won't make sense to write it for something very simple such as toggling classes based on state change, but for more advanced patterns like mixins it's vital. The answer to most questions about development is ["It depends!"](https://codepen.io/rachsmith/pen/YweZbG), this section embraces that. Here, we'll take an honest look at when the pattern is useful and when it should be avoided, or when something else makes more sense.
+Denna sektionen krävs inte, men rekommenderas starkt. Det är inte nödvändigt att skriva detta för något väldigt enkelt, t.ex. växlande klasser baserat på tillståndsförändringar, men för mer avancerade mönster så som mixins är det viktigt. Svaret på de flesta utvecklingsfrågor är ["Det beror på!"](https://codepen.io/rachsmith/pen/YweZbG), denna sektionen omfattar det. Här tar vi en ärlig titt på när mönstret är användbart och när det bör undvikas, eller när något fullständigt annat bör övervägas.
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: `Denna sektionen` :arrow_right: `Denna sektion` (2x)

Största delen av landet gör såhär:

1. denna/dessa/detta substantivet-i-obestämd-form
2. den/de/det här/där substantivet-i-bestämd-form

File: src/v2/cookbook/index.md [hunk 10]
```diff
@@ -1,91 +1,91 @@
...
+Denna sektionen krävs då du skrivit sektionen ovan om undvikande. Det är viktigt att utforska andra metoder så att folk som blivit tilsagda att något bör undvikas i vissa situationer inte lämnas förundrade. Tänk dig att webben är ett stort tält och att många människor har olika kodbasstrukturer som löser olika mål. Är appen stor eller liten? Integrerar de Vue i ett befintligt projekt, eller byggs det från grunden? Försöker deras användare endast nå ett eller flera mål? Finns det mycket asynkron data? Alla dessa problem påverkar alternativa implementeringar. Ett bra kokbokrecept ger utvecklare det här sammanhanget.
```
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: `Denna sektionen` :arrow_right: `Denna sektion`

File: src/v2/cookbook/index.md (hunk 1 above)
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: kunnskap :arrow_right: kunskap

Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: `istället` :arrow_right: `istället` (`i stället` ska särskrivas, inte sammanskrivas)

File: src/v2/cookbook/index.md (hunk 2 above)
Comment (sustained) [inline src/v2/cookbook/index.md]: :question: `utrymme att utforska` - `utrymme för att utforska` ser kanske lite bättre ut?

File: src/v2/cookbook/index.md (hunk 4 above)
Comment (sustained) [inline src/v2/cookbook/index.md]: :question: `där du inkluderar ditt koncept` :arrow_right: varför inte bara `med ditt koncept`?

No big deal though, do as you wish!

File: src/v2/cookbook/index.md (hunk 9 above)
Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: `så som` :arrow_right: `såsom` - ska sammanskrivas

I honestly feel bad pointing out samman- and särskrivningar, sorry! But... just writing whatever I see!

File: src/v2/cookbook/index.md (hunk 10 above)
Comment (sustained) [inline src/v2/cookbook/index.md]: :question: `då` :arrow_right: `när` (se ovan, inte särskilt viktigt)

Comment (sustained) [inline src/v2/cookbook/index.md]: :bug: `förundrade` :arrow_right: `förundrande`

File: src/v2/cookbook/index.md (hunk 2 above)
Comment (sustained) [inline src/v2/cookbook/index.md]: :question: `till den grad det är övervägande` - känns som om det är mycket vanligare att se och höra "den grad __som__ (det) är", det vill säga att i allmänhet så verkar `det` tas bort i både tal och skrift hellre än `som`?

**Review by sustained — CHANGES_REQUESTED** [review]
Thanks for working on this with me!

I just reviewed this for typos/grammatical errors and such, which have been marked with a :bug:. Such cases need fixing imo.

There's also a small handful of things I'm unsure on and/or are perhaps just me nitpicking (sorry), those are marked by a :question:. Such cases perhaps don't **need* fixing per se, I'll leave that up to you.

**Comment by sustained** [comment]
I just felt like trying out the code review feature! I've never used it before. Sorry if this is too much but I just added anything that jumped out at me. Just tell me if I'm being too nitpicky and then in future reviews I will only point out anything I perceive to be either a spelling or grammar mistake.

**Comment by Pwntus** [comment]
Thank you for the review, and no you're not being nitpicky. Your concerns are totally justified and only helps to improve the text. I also realize more and more how I'm pulling in sentence structures from norwegian, which can become somewhat awkward in swedish :P 

I've made changes to each point, and will mark them as "resolved".

File: src/v2/cookbook/index.md (hunk 1 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Fixed, thanks!

File: src/v2/cookbook/index.md (hunk 2 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Fixed, thanks!

File: src/v2/cookbook/index.md (hunk 3 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Changed to `komplexa detaljer`, thanks!

File: src/v2/cookbook/index.md (hunk 4 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Changed to `starta vänligen med...`, thanks!

File: src/v2/cookbook/index.md (hunk 5 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Fixed, thanks!

File: src/v2/cookbook/index.md (hunk 4 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Very true! Fixed.

**Comment by sustained** [comment]
> Thank you for the review, and no you're not being nitpicky. Your concerns are totally justified and only helps to improve the text. I also realize more and more how I'm pulling in sentence structures from norwegian, which can become somewhat awkward in swedish :P
> 
> I've made changes to each point, and will mark them as "resolved".

Don't worry about it, your Swedish is really good (but then again it is essentially physically impossible for a Norwegian to _not_ be good at Swedish :sweat_smile: in much the same way that I've never studied Norwegian at all but can read 85% of practically any Norwegian newspaper article)!

In any case it probably makes sense for both of us to review each other, mother tongue notwithstanding; two pairs of eyes are always better than one.

File: src/v2/cookbook/index.md (hunk 9 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Fixed, thanks!

File: src/v2/cookbook/index.md (hunk 10 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Changed to `ifall` since the last section is conditional.

Comment (Pwntus) [reply src/v2/cookbook/index.md]: Fixed, thanks!

File: src/v2/cookbook/index.md (hunk 2 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: You're right, fixed!

File: src/v2/cookbook/index.md (hunk 7 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: I'll omit the word here, sounds better.

File: src/v2/cookbook/index.md (hunk 8 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Changed to `när`. Also changed the last sentence to `...helst genom exemplet på alternativa mönster.`

File: src/v2/cookbook/index.md (hunk 9 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: Fixed in four places.

File: src/v2/cookbook/index.md (hunk 6 above)
Comment (Pwntus) [reply src/v2/cookbook/index.md]: (o)lämplig is better :+1:

**Comment by Pwntus** [comment]
All points have been addressed and updated.

**Pull request merged**